features/ui/loading-spinner/loading-spinner.tsx

```diff
@@ -0,0 +1,15 @@
+import Image from "next/image";
+import styles from "./loading-spinner.module.scss";
+
+export function LoadingSpinner() {
+  return (
+    <div data-cy="loading-spinner" className={styles.loadingSpinner}>
+      <Image
+        src="/icons/loading-spinner.svg"
+        width={64}
+        height={64}
```

Comment:
Interesting, I didn't notice that the svg has 66px as width and height but the design requires 64px.